Makefile

```diff
@@ -54,7 +54,13 @@ ifeq ($(GO),)
   $(error Could not find 'go' in path.  Please install go, or if already installed either add it to your path or set GO to point to its directory)
 endif
 
-export GOARCH ?= amd64
+MACH ?= $(shell uname -m)
+
+ifeq ($(MACH), ppc64le)
+   export GOARCH ?= ppc64le
+else
+   export GOARCH ?= amd64
```

Comment:
For Linux a MACH of "x86_64" would need to be set to "amd64".  I didn't check what a Mac reports.

Reply:
my mac says:
```
$ uname -m
x86_64
```

Reply:
@mattdelco so, this change should be fine, right? As @ldemailly reported, that a mac shows `x86_64` and I think is consistent with the x86 servers too. The `GOARCH` will only be set to `ppc64le` only if `MACH==ppc64le` and if `GOARCH` wasn't set to anything else before. Or, do we need to set that specifically for `MACH==x86_64`?

Reply:
I suppose it's fine.  I think my original mindset was that the 'else' case was "export GOARCH ?= $(MACH)", which would be a problem, but as you point out that's not how it was implemented.  I suppose it could be:

<pre>ifeq ($(MACH), ppc64le)
   export GOARCH ?= ppc64le
else ifeq ($(MACH), x86_64)
   export GOARCH ?= amd64
else
   export GOARCH ?= $(MACH)
endif</pre>

but I don't have a preference--it might depend on what arm reports (it's the only other platform that comes to mind offhand that's still in production and has decent popularity).

Reply:
Yeah. I guess, I can add the `amd64` bits specifically.